frontend/public/vendor.scss

```diff
@@ -56,7 +56,14 @@
 @import '~patternfly-react-extensions/dist/sass/filter-side-panel';
 @import '~patternfly-react-extensions/dist/sass/properties-side-panel';
 @import '~patternfly-react-extensions/dist/sass/vertical-tabs';
+
```

Comment:
You can remove all the changes in this file as they are unnecessary/undesirable.